numpy/random/tests/test_random.py

```diff
@@ -255,6 +255,18 @@ def test_random_integers_max_int(self):
         desired = np.iinfo('l').max
         np.testing.assert_equal(actual, desired)
 
+    def test_random_integers_deprecated(self):
```

Comment:
@charris : Could you clarify your comment?  There seems to be a mixture of suggestions for this actual PR and suggestions of a greater infrastructural change.  Should I move my test into ```test_deprecations.py``` as a derived class of ```_DeprecationTestCase```?